Pinta.Core/Classes/Palette.cs

```diff
@@ -184,7 +175,7 @@ static List<Color> LoadColors (PaletteFormatManager paletteFormats, Gio.File fil
 			}
 		}
 
-		throw new Exception (errors.ToString ());
+		throw new PaletteLoadException (file.GetParseName ());
```

Comment:
We're losing the `errors` string which was built up from the failed attempts to load the palette - I think it would make sense to use that as the exception's message rather than the string that's currently in the `PaletteLoadException` constructor?